data-extraction/ingredients.json

```diff
@@ -0,0 +1,43 @@
+[
+
+{
+    "id": 1,
```

Comment:
My bad, could have phrased that better. "Add" would have been a good keyword to use.